ArduPlane/Parameters.h

```diff
@@ -54,6 +54,8 @@ class Parameters {
         k_param_landing,
         k_param_NavEKF3,
         k_param_BoardConfig_CAN,
+        k_param_landinggear,    // 9
```

Comment:
I'm guessing this should go into G2, but I'll let @tridge decide on that.

Reply:
Just found the first free spot :)

Reply:
We're leave a few open for emergencies. This is not an emergency. Please move to g2.